test/functional/esperanza_slash.py

```diff
@@ -18,12 +18,13 @@
     TxType,
     sign_transaction)
 from test_framework.util import (
-    sync_blocks,
-    connect_nodes,
-    disconnect_nodes,
+    assert_equal,
     assert_finalizationstate,
     assert_raises_rpc_error,
-    assert_equal,
+    connect_nodes,
+    disconnect_nodes,
+    generate_block,
```

Comment:
I haven't found this function in the util.py file. Where is it defined?

Reply:
Thank you for attentiveness! I've pushed missed changes.